plugins/outputs/newrelic/newrelic.go

```diff
@@ -108,18 +118,21 @@ func (nr *NewRelic) Write(metrics []telegraf.Metric) error {
 			case uint64:
 				mvalue = float64(n)
 			case float64:
-				mvalue = float64(n)
+				mvalue = n
 			case bool:
 				mvalue = float64(0)
 				if n {
 					mvalue = float64(1)
 				}
 			case string:
-				// Do not log everytime we encounter string
-				// we just skip
-				continue
+				// If we see a string send it as a tag so it can be parsed as an attribute, max 255 chars
+				tagLength := len(n)
+				if tagLength > 255 {
+					tagLength = 255
+				}
+				tags[mname] = n[:tagLength]
```

Comment:
This is a change in behavior compared to what is done currently. As we try to keep the output stable over releases you need to guard this with an option that keeps the old behavior by default. I.e. add an option `submit_string_values_as_tag` or similar and keep it set to false by default. Then wrap your code here with `if nr.StringsAsTags`.
If this is controversal, you maybe should split out this change to another PR so we do not hold-up the proxy part.

Reply:
Reverted to original